### PR TITLE
TA -2_Added the Java Code for an Auto Service app

### PR DESCRIPTION
**Pull request by katexgithub**
I have added a series of 4 classes in order to define a structure for an auto service entity. After that I have created some objects and populated them with details. In result, I have printed out all the details in the CLI.

### CODE REVIEW

File: src/main/java/com/katyat/classesandobjects/autoservicetask/Car.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    int km;
+    int productionYear;
+    double carPrice;
+    String gearType;
+    String make;
+    String model;
+    String fuelType;
+
+    Car(String makeParametruDeIntrare, String modelFormalParameter, int productionYear, String fuelType) {
+        make = makeParametruDeIntrare;
```
Comment (verginia) [inline src/main/java/com/katyat/classesandobjects/autoservicetask/Car.java]: missing "this." for "make" property

File: src/main/java/com/katyat/classesandobjects/autoservicetask/Garage.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.katyat.classesandobjects.autoservicetask;
+
+import java.util.Date;
+import java.util.List;
```
Comment (verginia) [inline src/main/java/com/katyat/classesandobjects/autoservicetask/Garage.java]: where are you using these 2 imports: Date and List?

File: src/main/java/com/katyat/classesandobjects/autoservicetask/Tool.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    boolean isElevator;
+    boolean isUsed;
+    long uniqueCode;
+
+    Tool(){
+        System.out.println("A fost creat un instrument despre care nu se stie nimic");
+    }
+
+    Tool(long uniqueCodeParam){
+        uniqueCode = uniqueCodeParam;
```
Comment (verginia) [inline src/main/java/com/katyat/classesandobjects/autoservicetask/Tool.java:15]: missing "this." for "uniqueCode" property

**Review by verginia — COMMENTED** [review]
Please take a look at my comments

File: src/main/java/com/katyat/classesandobjects/autoservicetask/Tool.java (hunk 3 above)
Comment (katexgithub) [reply src/main/java/com/katyat/classesandobjects/autoservicetask/Tool.java:15]: Andrei deleted "this." for "uniqueCode" and left only (uniqueCode = uniqueCodeParam;)

File: src/main/java/com/katyat/classesandobjects/autoservicetask/Garage.java (hunk 2 above)
Comment (katexgithub) [reply src/main/java/com/katyat/classesandobjects/autoservicetask/Garage.java]: automatically appears in my IntelliJ

File: src/main/java/com/katyat/classesandobjects/autoservicetask/Car.java (hunk 1 above)
Comment (katexgithub) [reply src/main/java/com/katyat/classesandobjects/autoservicetask/Car.java]: Done

**Review by verginia — APPROVED** [review]

**Review by RazvanTrufan — APPROVED** [review]

**Pull request merged**